### ISSUE TITLE
[$500] [HOLD for payment 2025-02-04] Feature - Allow user to create international bank account

### ISSUE THREAD

**Issue by MonilBhavsar** [Expensify/App#51082]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Tracking issue https://github.com/Expensify/Expensify/issues/421064
Implement this section of design doc https://docs.google.com/document/d/19ltca0iKwB00pReokjBf9rEHoi1u8y-2_x-82jMibho/edit?tab=t.0#bookmark=id.183ls0sldt64

**Version Number:** N/A
**Reproducible in staging?:** N/A
**Reproducible in production?:** N/A
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** N/A
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:** N/A
**Issue reported by:** N/A
**Slack conversation:** N/A

## Action Performed:
Break down in numbered steps
N/A
## Expected Result:
Describe what you think should've happened
N/A
## Actual Result:
Describe what actually happened
N/A
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?
N/A
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021895154298503613105</li>
        <li>Upwork Job ID: 1895154298503613105</li>
        <li>Last Price Increase: 2025-02-27</li>
    </ul>
</details>

**Comment by MonilBhavsar** [comment]
@shawnborton sorry for the ping. This is implementation for design doc and design is already approved by the team

**Comment by lschurr** [comment]
Should we have External added here @MonilBhavsar or is this already assigned and being worked on?

**Comment by MonilBhavsar** [comment]
Making this Daily again since nothing is blocking

**Comment by MonilBhavsar** [comment]
https://expensify.slack.com/archives/C06ML6X0W9L/p1730608158275369?thread_ts=1729597274.005059&cid=C06ML6X0W9L

**Comment by joekaufmanexpensify** [comment]
Moving this back to daily as this is the last issue blocking releasing this project, so we should try and finish it asap

**Comment by joekaufmanexpensify** [comment]
Pending PR from @shubham1206agra

**Comment by MonilBhavsar** [comment]
Same update ^

**Comment by MonilBhavsar** [comment]
Looks like we're too close to have PR in review @shubham1206agra?

**Comment by twisterdotcom** [comment]
Relabelling as Lauren is OOO now.

**Comment by lschurr** [comment]
What's the latest here @MonilBhavsar @shubham1206agra?

**Comment by MonilBhavsar** [comment]
https://github.com/Expensify/App/pull/52322 is in review

**Comment by MonilBhavsar** [comment]
Yep, the PR was reverted and @shubham1206agra is working on a followup. So payment is not due here yet.
Also, this is an #expense project and shubham is being paid for the entire project and not this issue. So no payment required here as of now.

**Comment by lschurr** [comment]
Ah, great thanks! Closing this one out.

**Comment by lschurr** [comment]
Ah, sorry about that. Reopening.

**Comment by lschurr** [comment]
Any update here @shubham1206agra? Should this be made Weekly?

**Comment by lschurr** [comment]
@shubham1206agra can you complete the checklist for this one?

**Comment by DylanDylann** [comment]
@MonilBhavsar I have not been in the project. This is a big PR and lots of polish issues but It is almost done (still have some issues are on progress), should I get paid here separately or could I review more issues on the project and get paid later

**Comment by lschurr** [comment]
Waiting on @MonilBhavsar - https://github.com/Expensify/App/issues/51082#issuecomment-2633092246

**Comment by lschurr** [comment]
Pinged in Slack: https://expensify.slack.com/archives/C01SKUP7QR0/p1738688839940249

**Comment by MonilBhavsar** [comment]
> still have some issues are on progress

Could you please link them here?

**Comment by DylanDylann** [comment]
Currently, there is only [this issue](https://github.com/Expensify/App/issues/55672) that is still open. All the rest issues is closed

cc @shubham1206agra

**Comment by MonilBhavsar** [comment]
Yes, I'll assign you that issue and we can pay you after that issue is closed. If that's fine for you?

**Comment by lschurr** [comment]
Yep, that's fine @MonilBhavsar

**Comment by lschurr** [comment]
Closing this without payment as payment will be handled here: https://github.com/Expensify/App/issues/55672

**Comment by joekaufmanexpensify** [comment]
Coming from https://github.com/Expensify/App/issues/55672#issuecomment-2688519223, reopening this issue to handle @DylanDylann's C+ payment for this feature here, as this is the main issue for the FE implementation. 

@lschurr gonna grab this one from you to handle payment.

**Comment by joekaufmanexpensify** [comment]
Summarizing the discussion [here](https://github.com/Expensify/App/issues/55672#issuecomment-2673621043), @DylanDylann and I agreed he should be due $500 for his C+ reviews for this feature, covering:

- His original review of https://github.com/Expensify/App/pull/52322
- The follow up PR, which fixed several bugs from the original: https://github.com/Expensify/App/pull/54798
- This additional follow up PR: https://github.com/Expensify/App/pull/56236

**Comment by joekaufmanexpensify** [comment]
@DylanDylann offer sent for $500!

**Comment by joekaufmanexpensify** [comment]
@DylanDylann $500 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
Upwork job closed.

**Comment by joekaufmanexpensify** [comment]
All set.
